modules/mongodb/charts.go

```diff
@@ -529,6 +571,31 @@ func (m *Mongo) removeDatabasesFromDBStatsCharts(newDatabases []string) {
 	}
 }
 
+// removeReplicaSetMember removes dimensions for not existing
+// replicate set members
+func (m *Mongo) removeReplicaSetMembers(newMembers []string) {
```

Comment:
We need to remove the values from "replSetDimsEnabled" in that function too.

Reply:
Actually, we need to check if an id is in the "replSetDimsEnabled" and delete it only if it is in there (same as we do in the "collectReplSetStatus").

Reply:
done!